### ISSUE TITLE
The upper and lower arm controls should follow elbow

### ISSUE THREAD

**Issue by lucianomunoz** [Plyrolith/rigify_bendy#7]
![image](https://user-images.githubusercontent.com/5134226/110249564-8cc84f80-7f6e-11eb-9eba-65cb644e129a.png)
Screen left is current rigify, screen right is blendify, even though I feel the control gained in yours is much nicer, the controls from the upper and lower arm should always follow the elbow one as it will be much faster for posing and also if that isnt working you lose the posibility to pin elbows or to use the elbow control to correct pops.

Great stuff tho, I'd love it if we could push this to be the official rigify way!

**Comment by Plyrolith** [comment]
Hey Luciano!
Thanks for you feedback.
In terms of these controls as TWEAKS I disagree (and specifically changed the design to behave in the current way).

The tweaks in the official Rigs behave inconsistently which is against my expectations as an animator.
A tweak should NEVER affect another tweak and always be the last instance to control its rig section.
Anything that affects multiple tweaks would be a control layer ON TOP of the tweaks.

Referring to your example: I always ended up frustrated tweaking arm shapes only to find that the elbow tweaks disrupted my previous adjustments.

That being said: I completely agree that these joint adjustment controls are missing!
I am planning on another revamp for limbs (with a top layer curve control for ALL tweaks at once, as well as being able to place the tweak segmentation yourself instead of automated same-size segments) - and I am definitely going to include the control you're looking for!

To stay updated check out the development branch ;)

All the best,
Tristan

**Comment by lucianomunoz** [comment]
fair enough, I usually find it easier like that, but I was going to propose that second option but didn't in the name of simplicity ! haha, but good stuff we're on the same page :)

Also check in the custom properties rubber tweak property is still added but doesn't seem to do anything :)


Do you think you'll do this setup improvements for the spine too?
